requirements-precommit.txt

```
@@ -0,0 +1,3 @@
+flake8==3.8.3
+pre-commit==2.7.1
+cmakelang==0.6.13
```

Use the latest versions or else we will get update PRs from dependabot next month. (We still might, if they update the packages.)
```suggestion
flake8==3.8.4
pre-commit==2.8.2
cmakelang==0.6.13
```